extension/intents/navigation/navigation.js

```diff
@@ -56,14 +56,16 @@ this.intents.navigation = (function() {
 
   this.intentRunner.registerIntent({
     name: "navigation.translate",
-    description: "Translate the given page to English, using Google Translate",
+    description: "Translate the given page to the chosen language, using Google Translate",
     match: `
-    translate (this |) (page | tab | article | site |) (to english |) (for me |)
```

Comment:
This pattern should stay in, and default to English (since the tool only understands English, this is an appropriate default).

Reply:
Fixed.